doc/config.md

```diff
@@ -38,6 +38,7 @@ Some linters are not enabled by default. Right now these linters are:
 
 - `:missing-docstring`: warn when public var doesn't have a docstring.
 - `:unsorted-required-namespaces`: warn when namespaces in `:require` are not sorted.
+- `:refer`: warn when there is **any** usage of `:refer` or `:refer-all` in your namespace requirements.
```

Comment:
```suggestion
- `:refer`: warn when there is **any** usage of `:refer` in your namespace requirements.
```

"Refer all" still uses the `:refer` keyword but uses `:all` as the value (e.g. `[clojure.test :refer :all]`).

Reply:
Whoops, i got confused with the name of our linter :-D  Thanks!